tests/integration/test_analysisd/test_signature_id/data/test_cases/cases_invalid_signature_id.yaml

```diff
@@ -0,0 +1,13 @@
+- name: Empty Signature ID - wazuh restarts - rule ignored
```

Comment:
```suggestion
- name: Empty signature ID - Rule ignored
```

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/3649/commits/cf8e8008bb6aaf5eaf95b75a4c64f3757fba2713